include/SH3/graphics/quad.hpp

```diff
@@ -0,0 +1,45 @@
+/** @file
+ *
+ *  Class to render a fullscreen 2D quad to the screen.
+ *
+ *  @copyright 2017 Palm Studios
+ *
+ *  @date 11-2-2017
+ *
+ *  @note To render a quad, we need to split the screen into two right-angled triangles, meaning we need 6 vertices for one quad.
+ *
+ *  @author Jesse Buhagiar
+ */
+#ifndef QUAD_HPP_INCLUDED
+#define QUAD_HPP_INCLUDED
+
+#include "GL/glew.h"
+#include "GL/gl.h"
+#include "glm/glm.hpp"
+
+#include <vector>
+
+namespace sh3_graphics
+{
+    struct quad final
+    {
+    public:
+        /**
+         *  Class constructor. Takes
+         */
+        quad(const std::vector<glm::vec3>& verts);
```

Comment:
Ahh I see. I'll have to look at Mikes scene loader (I believe I sent it to you, it'll be in the file SH3_Loader.hpp/cpp), it's a bit of a mess so it's slightly hard to read.

For the most part I think it'll just be triangles and triangle strips, I'm not too sure of the efficieny of using a quad.